src/test/scripts/helm_install.sh

```diff
@@ -81,10 +81,41 @@ fi
 # Run the chart's tests
 helm test "$PRODUCT_RELEASE_NAME" --logs -n "${TARGET_NAMESPACE}"
 
-# Install an ingress route to allow access to Bitbucket from outside the k8s cluster
-HELM_RELEASE_NAME=${PRODUCT_RELEASE_NAME} envsubst < "${PRODUCT_INGRESS_TEMPLATE}-${clusterType}.yaml" >$LOG_DOWNLOAD_DIR/ingress.yaml
-
-kubectl apply -n "${TARGET_NAMESPACE}" --filename $LOG_DOWNLOAD_DIR/ingress.yaml
+# params:
+# HELM_RELEASE_NAME
+function install_ingress_route() {
+  # Install an ingress route to allow access to product from outside the k8s cluster
+  HELM_RELEASE_NAME=${HELM_RELEASE_NAME} \
+      envsubst \
+      < "${PRODUCT_INGRESS_TEMPLATE}-${clusterType}.yaml" \
+      > ${LOG_DOWNLOAD_DIR}/ingress-${HELM_RELEASE_NAME}.yaml
+  kubectl apply -n "${TARGET_NAMESPACE}" --filename ${LOG_DOWNLOAD_DIR}/ingress-${HELM_RELEASE_NAME}.yaml
+}
+
+# params:
+# NODE - numeric id of the node for which service should be installed
+# SERVICE_NAME - the requested service name
+function install_service_expose() {
```

Comment:
let's rename this, or add a comment, to be clear that we're installing a backdoor service for each pod, purely for func testing purposes